src/sidebar/services/groups.js

```diff
@@ -190,6 +219,11 @@ function groups(
         store.loadGroups(groups);
         if (isFirstLoad && groups.some(g => g.id === prevFocusedGroup)) {
           store.focusGroup(prevFocusedGroup);
+        } else if (
+          isFirstLoad &&
+          groups.some(g => g.id === directLinkedGroup)
+        ) {
+          store.focusGroup(directLinkedGroup);
```

Comment:
This condition should take precedence over the one above, otherwise any user who has used the client at least once before in the current browser, and is still a member of the last group they selected, will see that group selected instead of the direct-linked one.